untrusted/quote_verify/src/lib.rs

```diff
@@ -87,8 +145,8 @@ impl Quote {
     // TODO strict DER requires an extra 0 prefix on the r value when the r
```

Comment:
Is this OK to leave as a TODO? What are the consequences of hitting this case in this context? Seems like we'll fail to verify the signature and thus fail the report, which seems like a significant issue to me.

Reply:
I think I'd at least like to see an issue for it. The problem with TODOs is they don't get tracked against, like, releases, so we can easily end up shipping without addressing them. My first-order reading of this TODO is "if we ship with this it will be Bad because things will apparently-randomly fail to validate", hence my concern.

Reply:
This specific instance should be addressed by #27.  I'm fine moving forward with TODO's needing an issue (or perhaps PR.? not sure on the PR part as they could get changed before merge, but so could an issues text.